### PR TITLE
v0.2.1

### PR DESCRIPTION
**Pull request by umarcor**
# New Features

* Add PR template.
* Add `pyproject.toml`.

# Changes

* Update `.gitignore`.
* Move sources from `pyIPXACT` to `pyEDAA/IPXACT`.
* Use pyTooling.Packaging for the setup and the documentation.

# Bug Fixes

* Bump the version.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -34,21 +34,20 @@
...
 	description="A Document-Object-Model (DOM) for IP-XACT files.",
-	long_description=long_description,
-	long_description_content_type="text/markdown",
-	url="https://github.com/Paebbels/pyIPXACT",
-	packages=setuptools.find_packages(),
+	gitHubNamespace=gitHubNamespace,
+	sourceFileWithVersion=packageInformationFile,
+	developmentStatus="pre-alpha",
 	classifiers=[
 		"License :: OSI Approved :: Apache Software License",
```
Comment (Paebbels) [inline setup.py:52]: Do not specify the license as classifier, it's done automatically.

Do not specify Python versions. It's done automatically.

Comment (umarcor) [reply setup.py]: Done. Out of curiosity, where are is list of python versions defined?

File: setup.py [hunk 2]
```diff
@@ -34,33 +34,26 @@
...
+	packageName=packageName,
 	description="A Document-Object-Model (DOM) for IP-XACT files.",
-	long_description=long_description,
-	long_description_content_type="text/markdown",
-	url="https://github.com/Paebbels/pyIPXACT",
-	packages=setuptools.find_packages(),
+	gitHubNamespace=gitHubNamespace,
+	sourceFileWithVersion=packageInformationFile,
+	developmentStatus="pre-alpha",
 	classifiers=[
```
Comment (Paebbels) [inline setup.py:51]: Oh I wasn't precise enough - sorry.

Actually none of the classifiers is needed:
https://github.com/pyTooling/pyTooling.Packaging/blob/main/pyTooling/Packaging/__init__.py?ts=2#L257

*pre-alpha is added by `developmentStatus`.

File: setup.py (hunk 1 above)
Comment (Paebbels) [reply setup.py]: Here: https://github.com/pyTooling/pyTooling.Packaging/blob/main/pyTooling/Packaging/__init__.py?ts=2#L256

3.6 ... 3.10 like in Actions.

File: setup.py (hunk 2 above)
Comment (umarcor) [reply setup.py:51]: Done.

File: setup.py (hunk 1 above)
Comment (umarcor) [reply setup.py]: Hmmm. The list of versions is overriden if provided. Is the list of classifiers overriden as well? Or are those appended to the default?

**Comment by Paebbels** [comment]
Pipeline is OK. anything else to do?

**Comment by umarcor** [comment]
> Pipeline is OK. anything else to do?

Fixing the usage of "automodule" in Sphinx, which is failing as a result of renaming the package and/or moving it to a subdir.

**Pull request merged**